cpp/src/gandiva/projector.cc

```diff
@@ -173,8 +177,7 @@ Status Projector::Make(SchemaPtr schema, const ExpressionVector& exprs,
   for (auto& expr : exprs) {
     ARROW_RETURN_NOT_OK(expr_validator.Validate(expr));
   }
-
-  ARROW_RETURN_NOT_OK(llvm_gen->Build(exprs, selection_vector_mode));
+  ARROW_RETURN_NOT_OK(llvm_gen->Build(exprs, selection_vector_mode, obj_cache)); // to use when caching only the obj code
```

Comment:
Place the comment one line above the code line it refers to, I think it is better to read than put in same line

Reply:
Removed, it is not necessary any more.